src/index.html

```diff
@@ -11,8 +11,72 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="“anonymous”"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!-- <h1>Product cards</h1> -->
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="iMac-icon"
+        class="card__image"
+      />
+
+      <a
+        href="#"
+        class="card__title"
+      >
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </a>
+
+      <a
+        href="#"
+        class="card__product-code"
+      >
+        Product code: 195434
+      </a>
+
+      <div class="stars">
+        <div class="stars stars--active">
```

Comment:
Add the modifier class
```suggestion
        <div class="stars stars--4">
```